thehive/app/org/thp/thehive/controllers/v1/Conversion.scala

```diff
@@ -251,6 +252,42 @@ object Conversion {
         .transform
     }
 
+  implicit class InputTaxonomyOps(inputTaxonomy: InputTaxonomy) {
+
+    def toTaxonomy: Taxonomy =
+      inputTaxonomy
+        .into[Taxonomy]
+        .withFieldComputed(_.namespace, _.namespace)
+        .withFieldComputed(_.description, _.description)
+        .withFieldComputed(_.version, _.version)
+        .withFieldConst(_.enabled, false) // TODO always false when importing a taxonomy ?
```

Comment:
What should be the default value for enabled ?

Reply:
Yes, I would import the disabled by default